lib/KDateRange/KDateInput.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+
+  <div>
+    <fieldset :aria-labelledby="legendText" class="date-input-fieldset" aria-describedby="date-desc" aria-live="polite">
+      <KTextBox
+        :ref="inputRef"
+        :value="value"
+        type="text"
+        :label="legendText"
+        autoComplete="off"
+        :invalid="Boolean(errorMessage)"
+        :showInvalidText="Boolean(errorMessage)"
+        :invalidText="errorMessage"
+        @input="handleInput"
+      />  
+      <input type="hidden" name="date" :value="valueAsDate" data-test="valueAsDate">
+      <span class="k-date-vhidden">
+        <span v-if="valueAsDate" id="date-desc">
```

Comment:
`id="date-desc"` :eyes: -- What happens if there are multiple `KDateInput` components on the same page?

Reply:
Good point, I remember now that we did talk about a solution that generates unique ids. I will go back and add that.